resources/dicts/thoughts/alive/deputy.json

```diff
@@ -306,11 +305,8 @@
         ]
     },
     {
-        "id": "story_deputy_apprentice",
+      "id": "story_deputy_to_apprentice",
         "thoughts": [
-            "Is recalling an old story that might inspire the Clan in tough times",
-            "Feels inspired to craft a new legend based on recent events",
-            "Uses the stories of old to help guide {PRONOUN/m_c/self} decisions",
```

Comment:
Same question here

Reply:
Yep - intentional. These ones lived in the "story deputy to apprentice" tag, but they also lived in basic story deputy. Since they don't explicitly identify apprentices, nor particularly require them, I removed them from the apprentice variant.